src/rtl/hbregex.c

```diff
@@ -260,7 +260,8 @@ static HB_BOOL hb_regex( int iRequest )
    }
    pRegEx = hb_regexGet( hb_param( 1, HB_IT_ANY ),
                          ( ! hb_parldef( 3, 1 ) ? HBREG_ICASE : 0 ) |
-                         ( hb_parl( 4 ) ? HBREG_NEWLINE : 0 ) );
+                         ( hb_parl( 4 ) ? HBREG_NEWLINE : 0 ) |
+                         ( hb_parl( 8 ) ? HBREG_DOTALL : 0 ));
```

Comment:
Thanks for help Viktor,
Actually I didn't know how to propertly use the Inline Modifier.
Adding (?s) works fine so adding the option is unnecessary.